scripts/installer.sh

```diff
@@ -165,6 +165,18 @@ main() {
 					VERSION="bullseye"
 				fi
 				;;
+			pika)
+				PACKAGETYPE="apt"
+				# All versions of PikaOS are new enough to prefer keyring
+				APT_KEY_TYPE="keyring"
+				# Older versions of PikaOS are based on Ubuntu rather than Debian
+				if [ "$VERSION_ID" -lt 4 ]; then
+					OS="ubuntu"
```

Comment:
do we not need to set `VERSION` in this case..?

Reply:
One could argue we shouldn't support these and should just make the script only accept version 4 and above since that's all we can verify 🤔

Reply:
from os-release man page:

```
       VERSION=
           A string identifying the operating system
           version, excluding any OS name
           information, possibly including a release
           code name, and suitable for presentation
           to the user. This field is optional.

           Examples: "VERSION=17", "VERSION="17
           (Beefy Miracle)"".

       VERSION_ID=
           A lower-case string (mostly numeric, no
           spaces or other characters outside of
           0–9, a–z, ".", "_" and "-") identifying
           the operating system version, excluding
           any OS name information or release code
           name, and suitable for processing by
           scripts or usage in generated filenames.
           This field is optional.

           Examples: "VERSION_ID=17",
           "VERSION_ID=11.04".
```

so at least we shouldn't be relying on the `VERSION` in os-release, we should derive the version we need from more parseable fields.

I also think supporting ≥4 is reasonable. maybe we can consider older versions is folks ask

Reply:
Ah, looking at the other ubuntu-specific code it seems we can do `VERSION="$UBUNTU_CODENAME` to get what we need

Reply:
sounds good!